tools/online_editor/src/index.ts

```diff
@@ -60,15 +60,16 @@ function create_style_menu(editor: EditorWidget): Menu {
   for (const style of [
     { label: "Fluent", name: "fluent" },
     { label: "Fluent Light", name: "fluent-light" },
-    { label: "Fluent Dark", name: "fluent-dark" }]) {
+    { label: "Fluent Dark", name: "fluent-dark" },
+  ]) {
     const command_name = "slint:set_style_" + style.name;
     commands.addCommand(command_name, {
       label: style.label,
       isToggled: () => {
         return editor.style === style.name;
       },
       execute: () => {
-        return editor.style = style.name;
+        return (editor.style = style.name);
```

Comment:
I think this conflicts with my cleanup in master. But when rebasing you can just ignore this hunk. The return statement is bogus and shouldn't be there :)